manuscript/1-SoftReq.txt

```diff
@@ -77,134 +121,466 @@
 
```

Comment:
Πρόσθεσε την πρώτη πρόταση "The topic breakdown for the Software 
Requirements KA is shown in Figure 1.1."
Για την εισαγωγή των εικόνων μπορείς να χρησιμοποιήσεις την παρακάτω δομή που χρησιμοποίησε και ο προηγούμενος author:
![Σχήμα 1.1 Κατανομή των θεμάτων στο γνωστικό πεδίο Απαιτήσεις λογισμικού.](images/1-SoftReq.png)